Swift/BRCrypto/BRCryptoSystem.swift

```diff
@@ -509,9 +509,11 @@ public final class System {
     /// `Network` there will be `SystemEvent` which can be used by the App to create a
     /// `WalletManager`
     ///
-    /// @Note: This should only be called one.
+    /// - Parameter applicationCurrencies: If the BlockChainDB does not return any currencies, then
+    ///     use `applicationCurrencies` merged into the deafults.  Appropriate currencies can be
+    ///     created from `System::asBlockChainDBModelCurrency` (see below)
     ///
-    public func configure () {
+    public func configure (withCurrencyModels applicationCurrencies: [BlockChainDB.Model.Currency]) {
```

Comment:
I wanted to be clear-is in the interface that the `applicationCurrencies` are backups for what the BDB provides.  And, I didn't want to publicly expose the `Currencies` initializer.

That is one of the things that I thought would cause trouble in the Java...

I think it will be moot when 'tokens' use FileService.